AK/Types.h

```diff
@@ -68,6 +68,8 @@ using __ptrdiff_t = __PTRDIFF_TYPE__;
 #    if defined(AK_OS_WINDOWS)
 using ssize_t = AK::Detail::MakeSigned<size_t>;
 using mode_t = unsigned short;
+using gid_t = unsigned int;
```

Comment:
IMHO - I should remove this from the PR.

Reply:
maybe... but this is a hack to call `create_session_temporary_directory_if_needed()` which needs these types.